### PR TITLE
Add manual for files to the docs

### PR DESCRIPTION
**Pull request by pedroripper**
This PR adds a manual for the different files that PSRI uses.
It also renames `initialize_study` to `load_study`

### CODE REVIEW

File: docs/Project.toml
Comment (pedromxavier) [inline docs/Project.toml:1]: Set compatibility as

```toml
[compat]
Documenter = "~0.27"
DocumenterDiagrams = "~0.27"
```

File: docs/src/examples/modification.md
Comment (pedromxavier) [inline docs/src/examples/modification.md:1]: Say "existing" instead of "old"

File: docs/src/file_types/file_diagram.md
Comment (pedromxavier) [inline docs/src/file_types/file_diagram.md:1]: Excelent diagrams

File: docs/src/file_types/pmd.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+# PMD
+
+> 💡 **Tip:** We have a syntax highlighter for .pmd files available on [Visual Studio's Marktetplace](https://marketplace.visualstudio.com/items?itemName=pedromxavier.psr-pmd)
+
+> ⚠ **Warning:** The syntax for PMD files is case-sensitive
```
Comment (pedromxavier) [inline docs/src/file_types/pmd.md:5]: Documenter.jl has a few built-in [admonition options](https://documenter.juliadocs.org/stable/showcase/#Admonitions). Let's use them!

```suggestion
!!! tip
    We have a syntax highlighter for .pmd files available on [Visual Studio's Marktetplace](https://marketplace.visualstudio.com/items?itemName=pedromxavier.psr-pmd)

!!! warning
    The syntax for PMD files is case-sensitive 
```

File: docs/src/file_types/psrclasses.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        "Generation(2)": [
+            5.0,3.0
+        ],
+        "code":1
+    }
+}
+
+```
+
+> ‼ **Important:** Note that in the `SecondCollection` there are two parameters `Generation(1)` and `Generation(2)`. They represent the values for the parameter `Generation` in two different dimensions.
```
Comment (pedromxavier) [inline docs/src/file_types/psrclasses.md:34]:
```suggestion
!!! note
    In the `SecondCollection` there are two parameters `Generation(1)` and `Generation(2)`. They represent the values for the parameter `Generation` in two different dimensions.
```

File: docs/src/file_types/relation_mapper.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+In the example below, we have a Relation Mapper file with the following information:
+- The model `CustomModel` has two relations defined, one with `SecondCollection` and another with `ThirdCollection`.
+    - The relation with `SecondCollection` is a `1_to_1` relation, and the parameter is called `system`. 
+    - The relation with `ThirdCollection` is a `1_to_N` relation, and the parameter is called `station`. 
+- The model `SecondCollection` has two relations with `FourthCollection`.
+    - The first relation with `FourthCollection` is a `FROM` relation, and the parameter is called `no1`.
+    - The second relation with `FourthCollection` is a `TO` relation, and the parameter is called `no2`.
+
+> 💭 **Reminder:** The relation parameter stores the `reference_id` from the element of the Target Collection. See [PMD manual](./pmd.md)
```
Comment (pedromxavier) [inline docs/src/file_types/relation_mapper.md:18]:
```suggestion
!!! info "💭 Reminder"
    The relation parameter stores the `reference_id` from the element of the Target Collection. See [PMD manual](./pmd.md)
```

**Review by pedromxavier — APPROVED** [review]
Thanks @pedroripper, this PR looks very good to me. 

I think it is very important to have a discussion about the diagrams added in our next dev meeting.
Since `initialize_study` was changed to `load_study`, it is important to mark these changes as breaking and bump the minor version.

**Review by raphasampaio — APPROVED** [review]

**Pull request merged**